examples/player/simid_player.js

```diff
@@ -358,7 +359,7 @@ class SimidPlayer {
     this.hideAdPlayer_();
     this.adVideoElement_.src = '';
     this.destroySimidIframe();
-    this.contentVideoElement_.play();
+    // this.contentVideoElement_.play();
```

Comment:
revert

Reply:
good catch commented this out when testing because file was too big to load